presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -688,6 +695,19 @@ void PrestoServer::run() {
   }
 }
 
+void PrestoServer::startLinuxMemoryChecker() {
+#ifdef PRESTO_ENABLE_LINUX_MEMORY_CHECKER
```

Comment:
We can move this to a function inside `LinuxMemoryChecker.h` and only enable it on the Linux platform.
That way, PrestoServer.cpp file is clean of any checks.

Reply:
Removed